### DISCUSSION TITLE
How can i set seed value of FastAI Neural Network in v0.3.1?

### DISCUSSION THREAD

**Discussion by berkeserol** [autogluon/autogluon#1552] [Q&A] [answered]
I have learned from a closed issue that 
"FastAI Neural Network is not seeded in v0.3.1 and is different every time. This is fixed in upcoming v0.4."
How can i set seed value of FastAI Neural Network in v0.3.1?
Or How can i remove it from the tabular predictor?

Thanks

**Comment by Innixma** [comment, marked as answer]
You can remove it by specifying `excluded_model_types=['FASTAI']` or editing the hyperparameters dictionary (this is shown in the tutorials). It is not possible to set seed for FastAI in v0.3.1.
